Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -551,6 +551,7 @@ private void RestoreActionStatesAfterReResolvingBindings(UnmanagedMemory oldStat
                 newActionState.pressedInUpdate = oldActionState.pressedInUpdate;
                 newActionState.releasedInUpdate = oldActionState.releasedInUpdate;
                 newActionState.startTime = oldActionState.startTime;
+                newActionState.bindingIndex = oldActionState.bindingIndex;
```

Comment:
Would this have any problems for the existing 1.8.0-pre.1 release ?
Do we need to hold back the release for this fix?

Reply:
Also is there a related bug regarding the assert, I can't find it ATM

Reply:
This PR is against ISX-1455. This is one of the bugs that prevented all the tests being enabled after project-wide actions was added.